lambda/index.js

```diff
@@ -1,40 +1,39 @@
-const exec = require('child_process').exec;
-const queue = require('queue');
-
-var processRecord = function(record, context) {
-  const child = exec('./ruby_wrapper ' + "'" +  JSON.stringify(record) + "' '" + JSON.stringify(context) + "'", (result) => {
-    context.done(result);
-  });
-
-  child.stdout.on('data', function (data) {
-    data.split("\n").forEach(function(x) {
-      console.log(x);
-    });
-  });
-  child.stderr.on('data', function (data) {
-    data.split("\n").forEach(function(x) {
-      console.log(x);
-    });
-  });
-};
+const exec = require('child_process').execFile;
```

Comment:
Sorry, this is my typo from the code snippet I sent -- `exec` should probably be `execFile` for clarity. The call below would have to change as well.